default.nix

```diff
@@ -28,6 +28,7 @@ let iosSupport = system == "x86_64-darwin";
               echo ${drv.version} >VERSION
               ./boot
             '' + drv.preConfigure or "";
+            patches = [];
```

Comment:
???

Order should be updated on the overlays so that we keep those ghc patches.

Reply:
I already merged that I thought!

Reply:
Can you add a comment on why you are doing this? I think this is surprising enough to be documented.

Reply:
Done!